docs/blog/2018-10-04-journey-to-the-content-mesh/index.md

```diff
@@ -0,0 +1,62 @@
+# Delivering Modern Website Experiences: The Journey to a Content Mesh
+
+The CMS was born to make content sites _possible_. Now, it’s being reimagined to make them _incredible_.
+
+In the early 2000s, publishing on the web was difficult. The CMS emerged as a single application to store content, build sites, and deliver them to users.
+
+Over time, tools in each category got better and more specialized, while the quality of monolithic enterprise CMS applications like Adobe Experience Manager and Sitecore has stayed roughly the same.
+
+Today, a website team running an e-commerce site can store product inventory in [Shopify](https://www.shopify.com/), product listings in [Salsify](https://www.salsify.com/) and reviews in [Bazaarvoice](https://www.bazaarvoice.com/). A team running a paywalled content site can create stories in [Wordpress](https://wordpress.org/), store video in [JWPlayer](https://www.jwplayer.com/), user data in [Auth0](https://auth0.com/), and subscription data in [Recurly](https://recurly.com/).
+
+When website teams want to add search, they can turn to [Algolia](https://www.algolia.com/); for payments, [Stripe](http://stripe.com); for analytics, [Segment](https://segment.com/); for A/B testing, [Optimizely](https://www.optimizely.com/); for personalization, [Evergage](https://www.evergage.com/).
+
+Teams are adopting these services because each is tailored to a specific use-case, while the generic enterprise CMS alternative is optimized for none.
+
+This series is intended to explain the emergence of a content mesh -- the infrastructure layer for a decoupled website. The content mesh stitches together content systems in a modern development environment while optimizing website delivery for performance.
+
+The previous generation of enterprise CMS offered a set of mediocre components; a generic content editing experience glued to out-of-date development and deployment environments. The emerging strategy for content management selects best-of-breed solutions, tailored to specific use-cases like e-commerce or blogging; it provides a modern framework for rapid iteration and it generates lightning fast websites out of the box.
```

Comment:
I wouldn't say CMSs' editing experience is mediocre — many of the content editing experiences in CMSs are extremely good — the result of decades of UX research & real-world testing etc. Honing in that they're generic and costly to modify to meet unique needs vs. specialized tools will resonate better. 

Development & deployment can definitely be mediocre — though even those are often very sophisticated and acceptable — I'd focus more on how cloud/serverless changes things e.g. by making everything cheaper, more scalable, more efficient.

Reply:
I agree it's good -- it's often however generic -- not tied to particular use cases like e-commerce or blogging. 

The thrust of this series isn't the technical hosting/deployment piece (serverless/cloud) -- that's another story which we tell elsewhere, like in the New Stack article. 

It's more persona-oriented, targeted at the content editor / developer / user experience.